python/ray/experimental/state/api.py

```diff
@@ -618,6 +623,7 @@ def get_node(
     )
 
 
+@PublicAPI(stability="alpha")
```

Comment:
Btw, should I take it out from the experimental folder? Or should I do when it becomes beta (thought the latter, but lmk if it is the former)

Reply:
imo you should move out of experimental.

Reply:
https://github.com/ray-project/ray/pull/27846/files -> it will be done here.